src/packages/cdk/src/app/website.ts

```diff
@@ -0,0 +1,105 @@
+import path from 'path';
+import * as cdk from 'aws-cdk-lib';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as s3deploy from 'aws-cdk-lib/aws-s3-deployment';
+import * as cloudfront from 'aws-cdk-lib/aws-cloudfront';
+import * as origins from 'aws-cdk-lib/aws-cloudfront-origins';
+import { Certificate } from 'aws-cdk-lib/aws-certificatemanager';
+import { Construct } from 'constructs';
+
+import { GraphweaverAppConfig } from './types';
+
+export class WebsiteStack extends cdk.Stack {
```

Comment:
I'm thinking for our stuff we just want to attach constructs to whatever stack they create, rather than actually owning the stack ourselves.

Reply:
This allows us to control the deletion of stacks independently. As the Website, API and Database stack could be deleted we will keep these as seperate stacks.